src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1199,12 +1199,19 @@
       #(not (or (empty? %) (kitchensink/string-contains? "[" %)))
       (reverse containment-path)))))
 
-(def store-resources-column? (atom false))
-(defn maybe-resources
-  [row-map]
-  (if @store-resources-column?
+(def enterprise-data? (atom false))
```

Comment:
Fwiw @mullr and I talked about this quite a bit and settled on *not* grouping all these things together.